convert2rhel/unit_tests/subscription_test.py

```diff
@@ -360,11 +360,11 @@ def test_restore_subman_uninstalled(self, caplog, monkeypatch, system_subscripti
 
 def test_install_rhel_subsription_manager(monkeypatch, global_backup_control):
     mock_backup_control = mock.Mock()
-    monkeypatch.setattr(global_backup_control, "push", mock_backup_control)
+    monkeypatch.setattr(subscription.backup.backup_control, "push", mock_backup_control)
 
     subscription.install_rhel_subscription_manager(["subscription-manager", "json-c.x86_64"])
 
-    assert mock_backup_control.push.called_once
+    assert mock_backup_control.call_count == 1
```

Comment:
I see you changed this many times. Are there any benefits of using `call_count == 1` instead of `called_once`?

Reply:
I think it is more clear that we are specifically saying we want to assert how many times it was called. It could be converted to called_once, but I think checking count is better to readability.

Reply:
Ok, thanks for explanation. I agree